package.json

```diff
@@ -24,7 +26,7 @@
     "format:all": "prettier --write \"**/*.{cjs,html,js,json,md,ts}\"",
     "lint": "npm run lint:sh && npm run lint:ts",
     "lint:sh": "bash ./scripts/shellcheck.sh",
-    "lint:ts": "npx tslint 'src/**/*.ts' -e '**/node_modules/**'",
+    "lint:ts": "eslint \"**/*.{js,ts}\"",
```

Comment:
Shall we include this as a step in CI pipeline as well? I mean the CI for every PR.

Reply:
`npm run lint` is already in the ci